lib/config_default.ts

```diff
@@ -200,21 +366,9 @@ export const config = {
     "AP121U",
     "D-Link DIR-615",
     "D-Link DIR-615 D",
-    "D-Link DIR-825",
```

Comment:
We will still need all those here

Reply:
i added the D-Link DIR-825 back
i dont know why it was gone.

maybe we should use the upstream branch except the EOL changes and the auto deploy i dont think we got other changes in our repo.

Reply:
The proper way would be to use the config instead of changing the default values here.

Reply:
@T0biii sorry, what I meant: We need both the full list of eol and deprecated, similar to what we had before. We have a slightly different firmware (with different/more packages) and hence have slightly different lists of supported devices in comparison to upstream.

Reply:
i added the devices to the config and the defaults are not touched anymore:
https://github.com/freifunkMUC/meshviewer/blob/new-main/config.json